README.md

```diff
@@ -94,6 +93,21 @@ spec:
       name: quayio-olemefer
 ```
 
+### `BuildRun`
+
+The resource `BuildRun` (`buildruns.dev/v1alpha1`) is the build process of a build definition which is executed in Kubernetes. 
+Please consider the following example:
+
+```yaml
+apiVersion: build.dev/v1alpha1
+kind: BuildRun
+metadata:
+  name: buildpack-nodejs-buildrun
+spec:
+  buildRef:
+    name: buildpack-nodejs-build
+```
+
```

Comment:
The section below needs to be changed a bit to look like:




```
$ kubectl get buildruns
NAME         STATUS
buildpacks   Running
```

and so on..